### ISSUE TITLE
Editor listed as an author from ORCID

### ISSUE THREAD

**Issue by adammwilson** [Impactstory/impactstory-tng#5]
When an editor is included in the list of people on ORCID it is added to the list of authors in the impact story JSON.  

For example, from ORCID:

`@article{Wilson_2016,doi = {10.1371/journal.pbio.1002415},url = {http://dx.doi.org/10.1371/journal.pbio.1002415},year = 2016,month = {mar},publisher = {Public Library of Science ({PLoS})},volume = {14},number = {3},pages = {e1002415},author = {Adam M. Wilson and Walter Jetz},editor = {Michel Loreau},title = {Remotely Sensed High-Resolution Global Cloud Dynamics for Predicting Ecosystem and Biodiversity Distributions},journal = {{PLOS} Biology}}`

And in impact story JSON:

`{
            "_title": "Remotely Sensed High-Resolution Global Cloud Dynamics for Predicting Ecosystem and Biodiversity Distributions", 
            "altmetric_id": "6410740", 
            "altmetric_score": 362.858, 
            "authors": "Adam M. Wilson, Walter Jetz, Michel Loreau", 
            "doi": "10.1371/journal.pbio.1002415", 
            "events_last_week_count": 9, 
            "genre": "article", 
            "id": "w8j6n5MrmV", 
            "is_oa_journal": true, 
            "is_oa_repository": false, 
            "is_open": true, 
            "is_open_new": true, 
            "journal": "PLoS Biol", 
...
`

In my opinion, editors should not be included in the list of authors (unless it is a book, perhaps?).

**Comment by hpiwowar** [comment]
Yup, great spot.  Will add it to list to fix, and let you know when fixed.  Thanks for telling us, and for the specific example!
